modules/custom/d_p/js/d_p_ckeditor.js

```diff
@@ -7,33 +7,46 @@
         d_p_ckeditor();
       });
 
-      function d_p_ckeditor_add_js(doc) {
-        var cssId = 'd_p_ckeditor';
-        if (!doc.getElementById(cssId)) {
-          // add css for cke_editable class in ckeditor
-          var head = doc.getElementsByTagName('head')[0];
-          var link = doc.createElement('link');
-          link.id = cssId;
-          link.rel = 'stylesheet';
-          link.type = 'text/css';
-          link.href = '/profiles/contrib/droopler/modules/custom/d_p/css/d_p_ckeditor.css';
-          head.appendChild(link);
-        }
-      }
-
+      // @todo: Rework this to better solution when it will be possible.
       function d_p_ckeditor() {
-        if (typeof CKEDITOR !== "undefined") {
-          for (let instanceName in CKEDITOR.instances) {
-            if (CKEDITOR.instances[instanceName].element.$.parentElement.classList.contains('d_p_ckeditor_centered')) {
-              if (typeof CKEDITOR.instances[instanceName].document !== 'undefined') {
-                d_p_ckeditor_add_js(CKEDITOR.instances[instanceName].document.$);
-              } else {
-                setTimeout(function () {
-                  d_p_ckeditor_add_js(CKEDITOR.instances[instanceName].document.$);
-                }, 500);
+        if (typeof Drupal.CKEditor5Instances !== "undefined") {
+          $(once('d_p_ckeditor_centered', '.d_p_ckeditor_centered', context)).each(function () {
+            let editor_id = $(this).find('textarea').attr('data-ckeditor5-id');
+            let editor = Drupal.CKEditor5Instances.get(editor_id);
+            let $editor_editable = $(this).find('.ck-editor__editable');
+
+            // Set default alignment to center.
+            $editor_editable.children().each(function () {
+              if (!$(this).hasClass('text-align-left') && !$(this).hasClass('text-align-right') && !$(this).hasClass('.text-align-justify')) {
+                $(this).addClass('text-align-center');
               }
-            }
-          }
+            });
+            editor.setData($editor_editable.html());
+
+            // Add text-align-left class to selected element.
+            $(once('open_alignment_dropdown', '.ck.ck-dropdown.ck-toolbar-dropdown.ck-alignment-dropdown')).on('click', function() {
```

Comment:
how to add some listener for the alignment button click event, I tried to use "editor.commands.get('alignment').on('change:value', (evt, name, value, oldValue) => {" but it fires also when user clicks different paragraphs inside the editor, unless there is some way to differentiate when it was fired from the alignment button and when from inside of editor.